api/src/main/java/org/loomdev/api/event/block/BlockEvent.java

```diff
@@ -0,0 +1,90 @@
+package org.loomdev.api.event.block;
+
+import org.jetbrains.annotations.NotNull;
+import org.loomdev.api.block.BlockPointer;
+import org.loomdev.api.block.BlockState;
+import org.loomdev.api.event.Cancelable;
+import org.loomdev.api.event.Event;
+import org.loomdev.api.event.EventCause;
+import org.loomdev.api.item.ItemStack;
+
+public interface BlockEvent extends Event  {
+
+    @NotNull
+    BlockPointer getPointer();
+
+    interface Break extends BlockEvent, Cancelable {
```

Comment:
Add the actual BlockState at the point of breaking, like mentioned before? or is there any reason this wasn't added?